src/server/rest/v1/service/UtilsService.ts

```diff
@@ -1418,6 +1456,39 @@ export default class UtilsService {
     return properties;
   }
 
+  public static async isAuthorizedOnChargingStation(tenant: Tenant, chargingStation: ChargingStation,
```

Comment:
To move there methods used by OCPP and REST to a new class in 'src/server' named CommonUtilsService.